src/libs/AxelarQueryAPI.ts

```diff
@@ -22,6 +22,22 @@ interface TranslatedTransferRateLimitResponse {
   outgoing: string;
   limit: string;
 }
+interface GMPExpressTransactionParameters {
+  isGMPExpressTransaction: boolean;
+  transferAmount: BigNumber;
+  destinationContractAddress: `0x${string}`;
+  evmTokenSymbol: string;
```

Comment:
```suggestion
  tokenSymbol: string;
```
Also generic to other ecosystems